src/sql/src/plan/query.rs

```diff
@@ -217,12 +232,30 @@ impl<'a> Fold<Raw, Aug> for NameResolver<'a> {
                 ResolvedObjectName {
                     id: Id::Global(gid),
                     raw_name: normalize::unresolved_object_name(raw_name).unwrap(),
+                    print_id: true,
                 }
             }
         }
     }
 }
 
+pub fn resolve_names_stmt(
+    scx: &StatementContext,
+    query: Statement<Raw>,
+) -> Result<Statement<Aug>, anyhow::Error> {
+    // DNM: I think this is wrong!
+    let qcx = QueryContext::root(scx, QueryLifetime::OneShot);
```

Comment:
I think this might be what the comment above is re:, but this `qcx` looks unnecessary bc all you're getting out of it is the `scx.catalog`?

Reply:
Hrm forget why I had this originally, fixed!